Avoid Output Uninitialized Blobs in Load with load_all=1

chandlerzuo opened the pull request:
Summary:
When output blob names are specified while load_all=1, output blob names are ignored. However, this behavior is not documented. In this diff, we just disallow users to provide blob names when load_all=1.

See discussion at https://fb.workplace.com/groups/1405155842844877/permalink/2714909788536136/

Differential Revision: D14883698

dzhulgakov reviewed (commented):
looks like tests are failing

chandlerzuo commented:
Test failures have been fixed. Could you take another look? @dzhulgakov

chandlerzuo commented:
The failing job is because of lint errors in files that I actually did not touch, so I am not sure if I should address them.

chandlerzuo commented:
@pytorchbot rebase plz

chandlerzuo commented:
@pytorchbot rebase please @dzhulgakov Could you take another look?

chandlerzuo commented:
@pytorchbot rebase this

chandlerzuo commented:
@pytorchbot rebase this please

chandlerzuo committed:
Avoid Output Uninitialized Blobs in Load with load_all=1 (#19133)

Summary:
When output blob names are specified while load_all=1, output blob names are ignored. However, this behavior is not documented. In this diff, we just disallow users to provide blob names when load_all=1.

See discussion at https://fb.workplace.com/groups/1405155842844877/permalink/2714909788536136/
Pull Request resolved: https://github.com/pytorch/pytorch/pull/19133

Reviewed By: dzhulgakov

Differential Revision: D14883698

Pulled By: dzhulgakov

fbshipit-source-id: 0d92a223264045273290b2268b740c3ebc236259
